core/db/migrate/20210120142527_ensure_default_locale_in_spree_stores.rb

```diff
@@ -0,0 +1,5 @@
+class EnsureDefaultLocaleInSpreeStores < ActiveRecord::Migration[6.0]
+  def change
+    Spree::Store.where(default_locale: nil).update_all(default_locale: I18n.locale)
```

Comment:
@damianlegawiec it's valid to check before update if that table exists in DB ?

Reply:
@przemosk table and column are created in previous migrations so no need IMHO